pkg/tekton/pipelines.go

```diff
@@ -293,17 +293,20 @@ func CreateOrUpdatePipeline(tektonClient tektonclient.Interface, ns string, crea
 	return answer, nil
 }
 
-// PipelineResourceName returns the pipeline resource name for the given git repository, branch and context
-func PipelineResourceName(gitInfo *gits.GitRepository, branch string, context string, pipelineType PipelineType) string {
-	organisation := gitInfo.Organisation
-	name := gitInfo.Name
+// PipelineResourceNameFromGitInfo returns the pipeline resource name for the given git repository, branch and context
+func PipelineResourceNameFromGitInfo(gitInfo *gits.GitRepository, branch string, context string, pipelineType PipelineType) string {
+	return PipelineResourceName(gitInfo.Organisation, gitInfo.Name, branch, context, pipelineType)
+}
+
+// PipelineResourceName returns the pipeline resource name for the given git org, repo name, branch and context
+func PipelineResourceName(organisation string, name string, branch string, context string, pipelineType PipelineType) string {
 	dirtyName := organisation + "-" + name + "-" + branch
 	if context != "" {
 		dirtyName += "-" + context
 	}
 
 	if pipelineType == MetaPipeline {
-		dirtyName += "-" + pipelineType.String()
+		dirtyName = pipelineType.String() + "-" + dirtyName
```

Comment:
I was undecided on this one. The reason I in the end but it in the middle was, that if you do something like `kubectl get pipelines` you get the meta and execution pipeline "next" to each other due to the alphabetical order. Now you get the build and meta pipelines "grouped" together. 

The big drawback, of course, being that due to truncation it was hard to differentiate.